graphix/pattern.py

```diff
@@ -1287,7 +1316,8 @@ def minimize_space(self) -> None:
             self.standardize()
         meas_order = None
         if not self._pauli_preprocessed:
-            meas_order = self.get_measurement_order_from_flow()
+            cf = self.extract_opengraph().find_causal_flow()
```

Comment:
Note that this change preserves bug #157, but I think fixing it would be outside the scope of this PR. Ideally, we would like `cf = self.try_extract_causal_flow()`, but this may cause some regressions: `minimize_space` will be correct in the sense that it will preserve runnability, but in some examples the minimized pattern may use more space than the current heuristics.